pbs_nodefile.c

```diff
@@ -161,7 +162,7 @@ int slurm_spank_task_init(spank_t sp, int ac, char **av)
     }
     slurm_debug("spank: pbs_nodefile: SLURM_TASKS_PER_NODE: %s", taskcount);
 
-    written = snprintf(command + pos, len - pos - 1,"' SLURM_TASKS_PER_NODE='%s' %s", taskcount, generate_pbsnodefile_command);
+    written = snprintf(command + pos, len - pos - 1,"' SLURM_TASKS_PER_NODE='%s' perl -I /usr/lib64/perl5/5.32/ %s", taskcount, generate_pbsnodefile_command);
```

Comment:
is this portable EL8 vs EL9?
i guess this is pointing to another issue

Reply:
i suspect this can be fixed in the generate_pbsnodefile_command itself

Reply:
No need for el8 changes, they have a difference plugin version anyway.

Reply:
not in quattor at least

Reply:
imho, just add it; it has 0 risks (otoh, fixing it later is also easy)